jmix-flowui/flowui-kit/src/main/java/io/jmix/flowui/kit/meta/component/StudioComponents.java

```diff
@@ -2116,6 +2116,7 @@ public interface StudioComponents {
                     @StudioProperty(xmlAttribute = "required", category = StudioProperty.Category.VALIDATION, type = StudioPropertyType.BOOLEAN,
                             defaultValue = "false"),
                     @StudioProperty(xmlAttribute = "tabIndex", type = StudioPropertyType.INTEGER),
+                    @StudioProperty(xmlAttribute = "trimEnabled", type = StudioPropertyType.BOOLEAN),
```

Comment:
what about default value?

Reply:
This will be implemented later in the studio issue. Since the value depends on the application property, it is necessary to implement a new strategy at the meta-annotation level.